src/main/java/org/opentripplanner/model/Station.java

```diff
@@ -63,9 +63,28 @@ public Station(
     this.url = url;
     this.timezone = timezone;
     this.priority = priority == null ? DEFAULT_PRIORITY : priority;
-    this.geometry = computeGeometry(coordinate, childStops);
+    // Initialize the geometry with an empty set of children
+    this.geometry = computeGeometry(coordinate, Set.of());
+  }
+
+  /**
+   * Create a minimal Station object for unit-test use, where the test only care about id, name and
+   * coordinate. The feedId is static set to "F"
+   */
+  public static Station stationForTest(String idAndName, double lat, double lon) {
```

Comment:
I have seen a couple of these methods and would vote to move them to a test utility, perhaps call it `TestUtils`?

Reply:
The problem with a TestUtils is that become a HubAndSpoke wheels with dependencies to the entire OTP codebase, so I do not want that. I would like us to keep these "xyzForTest" factory methods until we have proper builders for the OTP internal model.

You can also create a TestUtil for each model package, but then it become difficult to know where to find the factory methods. 

The right way to solve this is to make builders or mutable/immutable versions. Many developers say that the main code should not contain test code, but I think of the test code as equally important to the main code and that the main code also needs to serve the test code. I think a system become much easier to maintain if the main code is designed to be unit-testable. Of cause, we try to limit the amount of special test code. But having a few test factory methods to create valid domain object is very useful. Note, these factor method should only be used when the domain object created play a supporting role in the test - it is not under test.